models/dydx/ethereum/dydx_ethereum_flashloans.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+      alias='flashloans'
+      , materialized = 'view'
+      , file_format = 'delta'
+      , unique_key = ['blockchain', 'tx_hash', 'evt_index']
+      , post_hook='{{ expose_spells(\'["ethereum"]\',
+                                  "project",
+                                  "dydx",
+                                  \'["hildobby"]\') }}'
+  )
+}}
+
+WITH flashloans AS (
+    SELECT d.evt_block_time AS block_time
+        , d.evt_block_number AS block_number
+        , CAST(json_extract(json_extract(d.update, '$.deltaWei'), '$.value') AS DECIMAL(38,0)) AS amount_raw
+        , d.evt_tx_hash AS tx_hash
+        , d.evt_index
+        , CASE WHEN MIN(CAST(json_extract(json_extract(d.update, '$.deltaWei'), '$.value') AS DECIMAL(38,0)) - CAST(json_extract(json_extract(w.update, '$.deltaWei'), '$.value') AS DECIMAL(38,0))) < 0 THEN 0
+            ELSE MIN(CAST(json_extract(json_extract(d.update, '$.deltaWei'), '$.value') AS DECIMAL(38,0)) - CAST(json_extract(json_extract(w.update, '$.deltaWei'), '$.value') AS DECIMAL(38,0))) END AS fee
```

Comment:
This modified sql is not able to run on dune sql engine yet due to some type issues.
https://dune.com/queries/2480384?d=11